examples/EventV1/EventV2.json

```diff
@@ -0,0 +1,26 @@
+        "type": "nae.pknca.input.data"
+      },
+      {
+        "id": "ProjectId:5e1f4591-9868-4663-9f32-524e37ccc1d2,ResourceId:cd956b5f-ab40-4fd5-aada-1cf04005b7bd",
+        "datacontenttype": "application/json",
+        "source": "{\"routeOfAdministration\":\"EV\",\"matrix\":\"Plasma\",\"method\":\"lin up/log down\",\"requiredPointsInTerminalPhase\":3,\"partialAreasCount\":3,\"partialAreasStartTimes\":[0,0,0],\"partialAreasStopTimes\":[5.1,6.3,13],\"ncaTimeField\":\"MRRELTM\",\"ncaConcentrationField\":\"ACONC\",\"uniquePkProfilesSortingVars\":[\"SUBJID\",\"TRTP\",\"AVISITN\"],\"finalOutputParams\":[\"Cmax\",\"AUClast\",\"Tlast\",\"Tmax\",\"Clast\"],\"minAdjustedR2\":0.8,\"aucExtrapolationCriterion\":40,\"profiles\":[{\"profileKeys\":{\"SUBJID\":\"foo\",\"TRTP\":\"foo1\",\"AVISITN\":\"foo3\"},\"startTime\":\"NA\",\"stopTime\":\"NA\"}]}",
```

Comment:
The 'source' type shouldn't be string, it should be object. Why escape the JSON to string?

Reply:
I made it a string because I figured your code was going to be getting a JSON string for the configuration and then parsing that string into a structure, based on the structure of the `EventV1.json` message structure. I'm happy to leave this an object, as it marginally simplifies my code.

Reply:
Great, I think we should just keep it as JSON as it simplifies both sides (double unescaping).

Reply:
I'm repeating this just to keep it on the last conversation thread:
Please see the CloudEvents Spec. The source is a reference to the context of the event, the data should go into the "data" field.

Reply:
1. Moved `source` to `data`. I updated my code in `projectx` and the sample here.
2. Changed from a JSON string to an object.